exchanges/graphcache/src/types.ts

```diff
@@ -136,22 +136,21 @@ type ResolverResult =
   | undefined;
 
 // Cache resolvers are user-defined to overwrite an entity field result
-export type Resolver = (
-  parent: Data,
-  args: Variables,
-  cache: Cache,
-  info: ResolveInfo
-) => ResolverResult;
+export type Resolver<
+  ParentData = Data,
```

Comment:
I'm actually thinking; this may also have to be `DataFields` and we made a mistake here? https://github.com/FormidableLabs/urql/blob/aa5685b390e733fe2a6f46655502ea31df0daded/exchanges/graphcache/src/operations/query.ts#L339